cmd/juju/status/output_tabular.go

```diff
@@ -201,18 +212,27 @@ func printApplications(tw *ansiterm.TabWriter, fs formattedStatus) {
 			w.Print(scale)
 		}
 
-		w.Print(app.CharmName, app.CharmChannel, app.CharmRev)
+		w.Print(app.CharmName, app.CharmChannel)
+		if app.CanUpgradeTo != "" {
+			w.PrintColor(output.WarningHighlight, app.CharmRev)
+		} else {
+			w.Print(app.CharmRev)
+		}
 		if fs.Model.Type == caasModelType {
-			w.Print(app.Address)
+			w.PrintColor(output.InfoHighlight, app.Address)
 		}
 
-		exposed := "no"
+		var exposed string
```

Comment:
I think that we can use "yes"/"no" directly as a function parameter. And we don't need the additional variable ('expose').